lib/project/results-view.js

```diff
@@ -96,18 +96,28 @@ class ResultsView {
   }
 
   getRowHeight(resultRow) {
+    let baseHeight;
+
     if (resultRow instanceof LeadingContextRow) {
-      return this.contextRowHeight
+      baseHeight = this.contextRowHeight
     } else if (resultRow instanceof TrailingContextRow) {
-      return this.contextRowHeight
+      baseHeight = this.contextRowHeight
     } else if (resultRow instanceof ResultPathRow) {
-      return this.pathRowHeight
+      baseHeight = this.pathRowHeight
     } else if (resultRow instanceof MatchRow) {
-      return this.matchRowHeight
+      baseHeight = this.matchRowHeight
     }
+
+    return this.getHeightAdjustedToFontSize(baseHeight)
+  }
+
+  getHeightAdjustedToFontSize(baseHeight) {
+    const lineHeight = 2;
```

Comment:
Value of line-height for items in project find results is equal to 2em. That's why we need to keep this multiplier when increasing/decreasing font size.